app/controllers/general.php

```diff
@@ -92,6 +92,13 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
         }
     }
 
+    if (array_key_exists('proxy', $project->getAttribute('apis', []))) {
```

Comment:
What is proxy? is this for custom domains/functions?

Reply:
Probably used for functions. Just copied it from the existing services code.